scripts/zones/Silver_Sea_route_to_Nashmau/Zone.lua

```diff
@@ -7,26 +7,41 @@ local ID = require("scripts/zones/Silver_Sea_route_to_Nashmau/IDs")
 -----------------------------------
 
 function onInitialize(zone)
-end;
+end
 
 function onZoneIn(player,prevZone)
-    local cs = -1;
-
-    return cs;
-end;
+    local cs = -1
+    -- Gets current vanadiel time and compares it with arrival time to start a timer to show a message before it docks.
+    local vHour = VanadielHour()
+    local vMin  = VanadielMinute()
+    local dockHour
```

Comment:
Should this have a = 0 just to be started and not a nil

Reply:
will set it as a zero and test out. thank you @KnowOne134  :cat: